tests/discover/manual/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "tmp=\$(mktemp -d)" 0 "Create tmp directory"
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    rlPhaseStartTest
+        rlRun "tmt run --id $tmp discover | tee output"
+        rlAssertGrep "0 tests selected" "output"
+        rlRun "rm output"
+    rlPhaseEnd
+
+    rlPhaseStartCleanup
+        rlRun "popd"
+        rlRun "rm -r $tmp" 0 "Remove tmp directory"
```

Comment:
When running the test, the cleanup phase gets stuck on this:

```
rm: remove write-protected regular file '/tmp/tmp.nU5eejyhTd/plans/default/discover/default/tests/.git/objects/70/ea47add39f7b679440993d77e48e3960917b7c'? 
```

Have you run into this as well? Shall we use `rm -rf` or `tmt run -r` instead?

Reply:
Hm, that's weird, it doesn't happen to me locally and looks like the tests in the CI pass too. AFAIK multiple tests use this kind of cleanup, temporary directory is created and used as `--id` and then removed. Lets try to use `tmt run -r`, should be safer and the test will be a bit cleaner. I will force push the change.

Reply:
Fixed, could you please check if it works now?